docs/usage.md

```diff
@@ -52,12 +50,105 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## Example workflow
+
+An example workflow for RNA-seq data from an experiment can look like this:
+
+1. Run the pipeline with `--qc_only` with default params to check the quality of your reads.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet.csv \
+  --outdir <OUTDIR> \
+  -profile docker \
+  --qc_only
+```
+
+2. Run the pipeline with `qc_only` and any custom params that you have decided to use based on your data.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet.csv \
+  --outdir <OUTDIR> \
+  -profile docker \
+  --qc_only \
+  --extra_fastp_args='--trim_front1 15 --trim_front2 15' \
+  --remove_ribo_rna \
+  -resume
+```
+
+3. Once you are happy with the quality of your reads, run the pipeline without `qc_only` to proceed to transcriptome assembly.
+   If you have a lot of samples, you may wish to only include one replicate per condition in the samplesheet for assembly, as this may be quicker to run. This command also runs transcriptome quality assessment steps and quantification.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet_assembly.csv \ # samplesheet with only one replicate per sample
+  --outdir <OUTDIR> \
+  -profile docker \
+  --extra_fastp_args='--trim_front1 15 --trim_front2 15' \
+  --remove_ribo_rna \
+  --extra_tr2aacds_args='-MINAA 50' \
+  --busco_lineage= 'arthropoda_odb10' \
+  -resume
+```
+
+4. If you created the transcriptome assembly with only one replicate per condition, you can now run the pipeline with the full samplesheet, path to `--transcript_fasta`, and `--skip_assembly` to quantify the expression of all samples. If you used a samplesheet with all samples in step 3, this step is not needed.
```

Comment:
something is off in this sentence after "full samplesheeet"

Reply:
Thanks, I've split it into two sentences.